tests/test_sync_canary.py

```diff
@@ -2,6 +2,7 @@
 from base import TestLinkedinAdsBase
 
 class LinkedinAdsSyncTest(TestLinkedinAdsBase):
+    """Test tap sync mode and metadata conforms to standards."""
```

Comment:
Aren't we covering this scenario in tests like all fields? Do we need this test?

Reply:
Yes, we covering this scenario in other test cases. Removed `test_sync_canary` test case.